src/test/rpc_tests.cpp

```diff
@@ -339,6 +339,29 @@ BOOST_AUTO_TEST_CASE(rpc_convert_values_generatetoaddress)
     BOOST_CHECK_EQUAL(result[2].get_int(), 9);
 }
 
+BOOST_AUTO_TEST_CASE(rpc_convert_named_values_generatetoaddress)
+{
+    UniValue result;
+    std::vector<std::string> valid_named_values = {
+        "nblocks=10",
+        "address=2N7xz1EnWubTbQmWFcVpCvT2RwKeDuzWr5g",
+        "maxtries=50"
+    };
+
+    BOOST_CHECK_NO_THROW(result = RPCConvertNamedValues("generatetoaddress", valid_named_values));
+    BOOST_CHECK_EQUAL(result[0].get_int(), 10);
+    BOOST_CHECK_EQUAL(result[1].get_str(), "2N7xz1EnWubTbQmWFcVpCvT2RwKeDuzWr5g");
+    BOOST_CHECK_EQUAL(result[2].get_int(), 50);
+
+    std::vector<std::string> invalid_named_values = {
+        "nblocks=10",
+        "address",
+        "maxtries"
```

Comment:
do we need this param for `RPCConvertNamedValues` to throw? It seems that `std::runtime_error` will be thrown when `address` is parsed.

Reply:
👍